cmd/deviceservice/deviceservice.go

```diff
@@ -18,6 +18,7 @@ import (
 	adddeviceservice "github.com/edgexfoundry-holding/edgex-cli/cmd/deviceservice/add"
```

Comment:
Do you want to get rid of these aliases as you did earlier such as adddeviceservice etc